README.md

```diff
@@ -5,8 +5,8 @@ FilmShelf is a web-based social application designed to connect users from aroun
 The Team:
 
 * Ibrahim Ivan (itzivan)
-* Ebaad Aziz 
-* Omair Hashmi
-* Syed Shahzaib Azmat
+* Ebaad Aziz ()
+* Omair Hashmi (ohashmi21)
+* Syed Shahzaib Azmat ()
```

Comment:
add shahzaibs github name which is sidesideangle

Reply:
Added please recheck